README.md

```diff
@@ -10,10 +10,10 @@ This gives you access to the reducer, constants, actions, and selectors availabl
 ## Add to your combineReducers()
 
 ```javascript
-import { reducer } from 'redux-waiter'
+import waiter from 'redux-waiter'
 
 combineReducers{
-  waiter: reducer
```

Comment:
Thanks Calvin.  Were you using it and ran into an issue?  The documentation seems correct to me as the name of the reducer is `waiter`.  

https://github.com/hixme/redux-waiter/blob/6718ab98002e161de90e07fdf4dab369cb54e06d/src/constants.js#L1

Reply:
Yeah, I did. Because the reducer is part of the default exported object and not a named export, that destructured import yields undefined.

Reply:
Right, thanks for the feedback